tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Install squashfs-tools to provide unsquashfs binary (Debian and RedHat)
+  package:
+    name: squashfs-tools
+    state: present
```

Comment:
Looks good to me @pescobar although can you add `become: True` so that the role can be invoked without requiring the caller to set this?